CHANGELOG.md

```diff
@@ -18,7 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 - Remove dependency to the ldap-mapper dogu in favour of direct LDAP connections (#99)
   - The vision of abstracting LDAP connections with help of the [ldap-mapper](https://github.com/cloudogu/ldap-mapper) dogu still remains. This change is an intermediate step until the necessary changes to the ldap-mapper dogu and the migration towards CAS 6 are completed.
-  
+- Remove dogu configuration key ```ldap/use_user_connection_to_fetch_attributes```. From now on, all connections to the LDAP to fetch user attributes are made via the system connection. (#103)
```

Comment:
While this works I'd like you to encourage to use triple Backticks (aka code fences) 
```
for 
multiline 
code 
pieces 
```
and single back for `inline code snippets`.